lib/tasks/curation_sny1_fix.rake

```diff
@@ -11,6 +11,11 @@ namespace :curate do
     pl_refs = PlantPopulation.pluck(:female_parent_line, :male_parent_line)
     pl_refs = pl_refs.flatten.uniq
     # This will throw an exception if not found
-    pl_refs.each{ |ref| PlantLine.find(ref) }
+    begin
+      pl_refs.each{ |ref| PlantLine.find(ref) }
+    rescue ActiveRecord::RecordNotFound => e
+      puts e.message
+      next
```

Comment:
Good point. Fixed. Also made it invulnerable for nil references, which will be introduced by purgatory rake tasks.